docs/brickify/table_handler.rst

```diff
@@ -0,0 +1,195 @@
+Consider this input file:
+
++--------+------------------+--------------------+----------+------+
+|VAV name|temperature sensor|temperature setpoint|has_reheat|thresh|
++========+==================+====================+==========+======+
+|A       | A_ts             | A_sp               | false    |  16  |
++--------+------------------+--------------------+----------+------+
+|B       | B_ts             | B_sp               | true     |  12  |
++--------+------------------+--------------------+----------+------+
+
+One of the things that can be a little tricky with the 'condition' operation is ensuring that the types are correct when crossing from CSV/TSV and into Python, especially for strings and Booleans.
+
+For example, this expression will fire for row A but not row B:
+
+:: 
+
+    conditions:
+      - |
+        {thresh} > 14
+
+Internally, this is converted to the string ``'16 > 14'`` and then passed to the Python ``eval()`` method, which returns ``True``. 
+
+A trickier version - which looks like our earlier example but is *slightly* different: 
+
+:: 
+
+    conditions:
+      - |
+        {has_reheat}
+
+In our example, this will fail! (Spoiler: we took away the quotes from our earlier example)
+
+The issue is that the has_reheat column is pulled in as a string, but is not valid Python because the capitalization of 'true' and 'false' is incorrect in the TSV file.
+
+One way to fix this is to correct the data:
+
++--------+------------------+--------------------+----------+------+
+|VAV name|temperature sensor|temperature setpoint|has_reheat|thresh|
++========+==================+====================+==========+======+
+|A       | A_ts             | A_sp               | False    |  16  |
++--------+------------------+--------------------+----------+------+
+|B       | B_ts             | B_sp               | True     |  12  |
++--------+------------------+--------------------+----------+------+
+
+This will match the condition because we have capitalized True and False. 
+Unfortunately, changing the data in the input CSV you are processing may not always be possible. 
+
+As a compromise, to support this common use case where the input strings look like booleans but are not quite formatted right, Brickify expects Boolean conditions to be handled first as quoted strings:
+:: 
+
+    conditions:
+      - |
+        '{has_reheat}'
+
+Brickify will pass that code to the Python ``eval()`` method, which will return ``'true'``, which is type ``str`` (and not ``True`` which is type ``Boolean``) 
+However, as a special case, Brickify converts the following strings to booleans: 
+["TRUE", "true", "True", "on", "ON"] all become ``True``, and ["FALSE", "false", "False", "off", "OFF"] are converted to ``False``.
+
+An important note: the replacement text is not carried out on the substrings. At present, this will **not** work:
+
+:: 
+
+    conditions:
+      - |
+        {thresh} > 12 and '{has_reheat}'
```

Comment:
Just wanted to mention that the `conditions` block here is a list and you could add all conditions as items in the list as well. Something like (I think):
```yaml
    conditions:
       - {thresh} > 12 
       - '{has_reheat}'
```

That would be logically equivalent to having just one condition like `<cond1> and <cond2>`